pkg/rtcp/source_description.go

```diff
@@ -0,0 +1,264 @@
+package rtcp
+
+import (
+	"encoding/binary"
+
+	"github.com/pkg/errors"
+)
+
+// RTP SDES item types registered with IANA. See: https://www.iana.org/assignments/rtp-parameters/rtp-parameters.xhtml#rtp-parameters-5
+const (
+	SDESEnd      = iota // end of SDES list                RFC 3550, 6.5
```

Comment:
Can we make these an enum (and have a short ToString() method)

Reply:
OK

Reply:
I'll do the same for the RTCP packet types